lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -286,12 +287,9 @@ def set_aspect(self, aspect, adjustable=None, anchor=None, share=False):
             'equalyz'   adapt the y and z axes to have equal aspect ratios.
             =========   ==================================================
 
-        adjustable : None
-            Currently ignored by Axes3D
-
-            If not *None*, this defines which parameter will be adjusted to
-            meet the required aspect. See `.set_adjustable` for further
-            details.
+        adjustable : {'datalim', 'box'}, default: 'datalim'
```

Comment:
I believe the default for 2D plots is 'box', I personally did datalim since that's the one I use the most but if both are going in there's an argument to be made for the consistency.

Reply:
Here: https://github.com/matplotlib/matplotlib/blob/65cbdefcc42db7253dc0c6a5b22d8e9179ad4594/lib/matplotlib/axes/_base.py#L628

Reply:
Sounds good! It's totally fair game to modify other tests if the changes are justified.